R/mod_hierarchical_count_table.R

```diff
@@ -485,28 +495,39 @@ hierarchical_count_table_server <- function(
     on_sbj_click_fun = function() NULL,
     default_hierarchy = NULL,
     default_group = NULL,
+    hierarchy_choices = NULL,
+    group_choices = NULL,
     intended_use_label = NULL) {
   mod <- function(input, output, session) {
     ns <- session[["ns"]]
 
     inputs <- list()
     inputs[[EC$ID$HIERARCHY]] <- col_menu_server(
       id = EC$ID$HIERARCHY, data = table_dataset,
-      label = "Event count by",
+      subjid_var = subjid_var,
+      label = shiny::div(shiny::tags$label("Event count by"),
```

Comment:
I did not do it before, but just because we are modifying it, maybe we could move this strings into the pack of constants?